src/Sfx/package.json

```diff
@@ -27,19 +24,31 @@
     "gulp-plumber": "1.1.0",
     "gulp-replace": "0.5.4",
     "gulp-sass": "^4.0.1",
-    "gulp-sourcemaps": "2.5.1",
+    "gulp-sourcemaps": "^2.5.1",
     "gulp-tslint": "7.1.0",
     "gulp-typescript": "3.1.5",
     "gulp-uglify": "2.1.2",
     "merge2": "^1.2.1",
+    "natives": "^1.1.3",
     "run-sequence": "1.2.2",
     "tslint": "4.5.1",
     "typescript": "2.2.1",
-    "wiredep": "4.0.0",
-    "yargs": "7.0.1",
-    "natives": "^1.1.3"
+    "yargs": "7.0.1"
   },
   "dependencies": {
+    "adal-angular": "1.0.9",
+    "angular": "1.5.0",
+    "angular-animate": "1.5.0",
+    "angular-aria": "1.5.0",
+    "angular-route": "1.5.0",
+    "angular-sanitize": "1.5.0",
+    "angular-ui-bootstrap": "2.5.6",
+    "bootstrap-sass": "3.3.7",
+    "d3": "3.5.17",
+    "jquery": "^2.2.3",
+    "jquery-ui": "^1.12.1",
+    "lodash": "^4.17.10",
+    "moment": "^2.8.3",
```

Comment:
Why these become the dependencies of SFX Standalone?

Reply:
They are not dependencies of SFX Standalone. However the dependency list in 3rd party notice includes both dependencies from SFX and SFX Standalone.